ChiTech/ChiMesh/UnpartitionedMesh/lua/create.cc

```diff
@@ -39,14 +41,18 @@ int chiUnpartitionedMeshFromVTU(lua_State* L)
 {
   const char func_name[] = "chiUnpartitionedMeshFromVTU";
   int num_args = lua_gettop(L);
-  if (num_args != 1)
+  if (num_args < 1)
     LuaPostArgAmountError(func_name,1,num_args);
 
   const char* temp = lua_tostring(L,1);
+  const char* field = "";
+  if (num_args >= 2) field = lua_tostring(L,2);
```

Comment:
should this be == rather than >= ?

Reply:
That's fine cause extra arguments should be ignored, not alter behavior. If a third argument is added by accident the >= will still work but not ==

Reply:
At the moment, there exists another method defined in the same file that accepts an optional argument and it employs the same logic as above; the line identified has been left as-is for consistency.